warrant/django/backend.py

```diff
@@ -67,7 +67,10 @@ def authenticate(self, username=None, password=None):
         :return: returns User instance of AUTH_USER_MODEL or None
         """
         cognito_user = CognitoUser(
-            settings.COGNITO_USER_POOL_ID,settings.COGNITO_APP_ID,
+            settings.COGNITO_USER_POOL_ID,
+            settings.COGNITO_APP_ID,
+            access_key=getattr(settings, 'COGNITO_AWS_ACCESS_KEY_ID', None),
```

Comment:
I'd like to make the settings AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY like most Django project so if you use the default environment variables it isn't so confusing.